.deepsource.toml

```diff
@@ -0,0 +1,31 @@
+version = 1
```

Comment:
I don't see why we should add whatever this file is
Please remove this file

Reply:
sure sure it was my working changes tht got committed here sorry